### ISSUE TITLE
Mongodb error 11000 during installation 

### ISSUE THREAD

**Issue by MoeidHeidari** [polonel/trudesk#453]
![Screenshot from 2021-11-07 23-48-59](https://user-images.githubusercontent.com/41738051/140661450-2e416ec2-fada-428a-b805-ca1cfbb89c7a.png)
**Is this a BUG REPORT or FEATURE REQUEST?**:

- [ x] BUG 
- [ ] FEATURE

**What happened**:
I got the error 11000 during the installation and right after making Admin user.
**What did you expect to happen**:
I expected the admin user to be created and redirect to dashboard
error:
```bash
{"success":false,"error":{"driver":true,"name":"MongoError","index":0,"code":11000}}
```
**How to reproduce it (as minimally and precisely as possible)**:
I am using trudesk in docker

**Anything else we need to know?**:

**Environment**:
- Trudesk Version: latest
- OS (e.g. from /etc/os-release):
- NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

- Node.JS Version:
- MongoDB Version: 3.6
- Is this hosted on cloud.trudesk.io: no (on docker)

**Comment by dessy4oko** [comment]
Enter the mongo shell then
`use trudesk`
and
`db.dropDatabase()`
Restart the installation and you probably will get another error about ValidatorError (Path value is required).
Go to ./src/controllers/install.js and at 177 line
`if(!eEnabled)) return next()` should be uncommented
Rebuild container if you are using docker.
Restrart installation. Go throw all steps till it finishes. Comment line 177 back in ./src/controllers/install.js
Now you should be able to log in.

**Comment by polonel** [comment]
This was fixed in [v1.1.11](https://github.com/polonel/trudesk/releases/tag/v1.1.11)
